ui/src/views/Quickstart/index.jsx

```diff
@@ -359,19 +359,24 @@ export default class QuickStart extends Component {
                 For independent developers and organization owners: How to set
                 up your repository with {window.env.APPLICATION_NAME}
               </Typography>
-              <Typography paragraph>
-                <ul>
-                  <li>
+
+              <ul>
+                <li>
+                  <Typography paragraph>
```

Comment:
Putting the Typography component inside the ```ul``` did the trick. The error is now gone. @helfi92

Reply:
That's a good solution!